extensions/ray-so/package.json

```diff
@@ -7,11 +7,11 @@
   "author": "garrett",
   "contributors": [
     "RSO",
-    "vette",
```

Comment:
About that 
> Linting fixes (one contributor not found)

Would you mind updating `vette` to `dgrzedzielski`? I've updated username on raycast

Reply:
Ahh sorry, fixed!😄

Reply:
No worries, not your fault, thank you so much for updating!